controllers/jenkinsconfig/agent_config_test.go

```diff
@@ -0,0 +1,173 @@
+package jenkinsconfig
```

Comment:
The tests failed.

![image](https://user-images.githubusercontent.com/1450685/125224675-a0ab0600-e300-11eb-800f-45bda28eedc3.png)

Reply:
Thank you for your patient review and test. Due to exclusion of controller tests, this test won't be executed by GitHub Action:

https://github.com/LinuxSuRen/ks-devops/blob/20c7136f6ac759e543c01251045626300452b9c0/Makefile#L24